packages/legend-art/style/base/_panel.scss

```diff
@@ -203,6 +203,10 @@
   .input--small {
     max-width: 45rem;
   }
+
+  .input--large {
```

Comment:
This one is meant to force it to have 45rem since if the element is under <PanelListItem> the width will automatically be small. But I will change naming from `--large` :D